CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## [Unreleased](https://github.com/idealista/nginx-role/tree/develop)
+### Added
+- *Added configurable limit no file* @javierRobledo, @danieljesus

```

Comment:
Just a tiny thing: could you please add the link to the PR in this line?

Reply:
It's better to create an issue explaining the requested feature/problem 😉